public/pages/CreateTrigger/containers/ConfigureActions/ConfigureActions.js

```diff
@@ -183,7 +183,10 @@ class ConfigureActions extends React.Component {
             description: '',
           }));
       } else {
-        backendErrorNotification(notifications, 'load', 'destinations', response.err);
+        // If the config index is not created, don't show the notification
+        if (response.totalMonitors !== 0) {
```

Comment:
Club the else if?

Reply:
Oh sorry, I missed a case here. I got to display the error toast if the response is not okay and the totalMonitors is not zero. Will add that.